File: FIND-DISSAPEARED/fd.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        vector<int> s;
        unordered_set <int> hs(nums.begin(), nums.end());
        for (int i = 1; i<= nums.size(); i++){
            if(hs.find(i) == hs.end()){
                s.push_back(i);
            }
        }
        return s;
    }
};

File: FIND-DISSAPEARED/fd.java
class Solution {
public List<Integer> findDisappearedNumbers(int[] nums) {
         Set<Integer> hs = new HashSet<>();
        for (int num : nums) {
            hs.add(num);
        }

        List<Integer> s = new ArrayList<>();
        for (int i = 1; i <= nums.length; i++) {
            if (!hs.contains(i)) {
                s.add(i);
            }
        }

        return s;
    }
};